client/app/core/rbac.service.js

```diff
@@ -45,19 +35,13 @@ export function RBACFactory (lodash) {
     return features
   }
 
-  function setNavFeatures (features) {
-    navFeatures = features
-  }
-
   function setRole (newRole) {
     currentRole = newRole
   }
 
-  function getNavFeatures () {
-    return navFeatures
-  }
-
-  function navigationEnabled () {
-    return lodash.some(navFeatures, (item) => item.show)
+  function suiAuthorized () {
```

Comment:
Ya know, while we're at it, why not just shove this function into `authorization.config.js`?  It doesn't so much belong in rbac service anyways

Reply:
Oh i guess the login state is using it as well 🤔

Reply:
Yeah, thats why i renamed it.  I also figured that defining what allows someone to get into SUI might change so it might make sense to have a small function that determines it.

Reply:
but for now this doesn't hurt